megalista_dataflow/data_sources/data_schemas.py

```diff
@@ -158,7 +158,7 @@
             {'name': 'user_property_\\d+', 'required': False, 'data_type': 'string'},  
         ],
         'groups': [
-            ['apps_instance_id', 'client_id']
+            ['app_instance_id', 'client_id']
```

Comment:
"firebase_app_id, measurement_id" may be added as another element within the groups - one of them needs to be defined during execution. And another for "is_event, is_user_property". 

Having them set within Data Schemas will validate the schema before reaching do_process.